cmd/optimizely/main_test.go

```diff
@@ -215,6 +217,7 @@ func TestViperEnv(t *testing.T) {
 	_ = os.Setenv("OPTIMIZELY_CLIENT_BATCHSIZE", "1")
 	_ = os.Setenv("OPTIMIZELY_CLIENT_QUEUESIZE", "10")
 	_ = os.Setenv("OPTIMIZELY_CLIENT_FLUSHINTERVAL", "1m")
+	_ = os.Setenv("OPTIMIZELY_CLIENT_DATAFILEURLTEMPLATE", "https://localhost/v1/%s.json")
```

Comment:
@mikecdavis 
Shouldn't it be `DATAFILE_URL_TEMPLATE`

Reply:
No this needs to be `DATAFILEURLTEMPLATE` based on how viper maps config values to ENV variables.